src/script/E2EIdentity/E2EIdentity.ts

```diff
@@ -54,12 +54,12 @@ class E2EIHandler {
   private gracePeriodInMS: number;
   private currentStep: E2EIHandlerStep | null = E2EIHandlerStep.UNINITIALIZED;
 
-  private constructor({discoveryUrl, gracePeriodInMS}: E2EIHandlerParams) {
+  private constructor({discoveryUrl, gracePeriodInSeconds}: E2EIHandlerParams) {
     // ToDo: Do these values need to te able to be updated? Should we use a singleton with update fn?
     this.discoveryUrl = discoveryUrl;
-    this.gracePeriodInMS = gracePeriodInMS;
+    this.gracePeriodInMS = gracePeriodInSeconds * 1000;
```

Comment:
good to use 1000 as const

Reply:
There is a util for that. Look for `TIME_IN_MILLIS.SECOND` ;)